TaskLayer/CalibrationTask/CalibrationTask.cs

```diff
@@ -105,18 +105,18 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
 
                     if (i == 1) // failed round 1
                     {
-                        CommonParameters.PrecursorMassTolerance = new PpmTolerance(20);
```

Comment:
Is this the main change?

Reply:
I just read the description, and it looks like the answer is yes. :)

Reply:
Yep. When calibration fails the tolerance doesn't get reset for the next file.